DESCRIPTION

```diff
@@ -19,7 +19,7 @@ Depends:
     fabricatr (>= 0.5.0),
     estimatr (>= 0.10.0)
 Imports:
-    rlang, methods
+    rlang
```

Comment:
Did you ever figure out why methods was needed in the first place?

Reply:
I think there was an old importFrom call that was dropped at some point. Got an error in some builds on CRAN from this in the last version (all linux), but didn't seem to bother other builds.